wire-ios-sync-engine/Source/SessionManager/SessionManager+APIVersionResolver.swift

```diff
@@ -23,6 +23,10 @@ private let log = ZMSLog(tag: "APIVersion")
 extension SessionManager: APIVersionResolverDelegate {
 
     public func resolveAPIVersion(completion: @escaping (Error?) -> Void = { _ in }) {
+        guard !DeveloperFlag.multibackend.isOn else {
+            completion(nil) // we don't need to resolve apiversion here
+            return
```

Comment:
@johnxnguyen  since multibackend support is on there is no need to resolve the api here, no?

Reply:
correct, with multibackend api resolving happens during session loading.